code/__DEFINES/tgs.dm

```diff
@@ -0,0 +1,202 @@
+//tgstation-server DMAPI
```

Comment:
How do you add a file that already exists?

Reply:
aaaaAAAAA